### PR TITLE
--parallel-suites documentation & path fix

### PR DESCRIPTION
**Pull request by matthewnessworthy**

### CODE REVIEW

File: src/Runners/PHPUnit/FullSuite.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ class FullSuite extends ExecutableTest
      */
     public function __construct($suiteName, $configPath)
     {
-        parent::__construct($suiteName);
+        parent::__construct('');
```
Comment (andreasschroth) [inline src/Runners/PHPUnit/FullSuite.php]: What is this change for?

**Comment by andreasschroth** [comment]
Thank you for your contribution, what is the change in the constructor call for?

**Comment by matthewnessworthy** [comment]
the constructor was incorrectly a double path

**Comment by andreasschroth** [comment]
@matthewnessworthy Yes, right, thanks for the fix and the added documentation. 👍 

PS: Greetings to Gozo from St. Pauls Bay.

**Comment by matthewnessworthy** [comment]
👋 what are the chances :)

**Comment by Slamdunk** [comment]
Test added in https://github.com/paratestphp/paratest/pull/494

**Pull request merged**